src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -2183,15 +2108,11 @@ protected String repeatThis(String s, int n) {
         return  String.format(String.format("%%0%dd", n), 0).replace("0",s);
     }
     
-    //KFSPTS-1460
     public AchBundlerHelperService getAchBundlerHelperService() {
         return achBundlerHelperService;
```

Comment:
It looks like cu-spring-pdp.xml is already injecting this service.  I am not sure why spring context was used above.  Is there a way to test this locally? I am wondering if there was a spring mapping problem that was worked around by using spring context above.